### PR TITLE
#60810421 Return all necessary fields from the API

### PR DESCRIPTION
**Pull request by Veraclins**
#### What does this PR do?
- Return all the necessary fields from the API when getting a resource

#### Description of Task to be completed?
- modify the return queries to include all the necessary fields.
- add more seeds to the database to display on the frontend.

#### How should this be manually tested?
- Clone the repo
- Run `npm install` 
- Run `npm start` or `npm run dev`
- test the different get requests with postman.

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#60810421](https://www.pivotaltracker.com/story/show/60810421)

#### Questions:
- None

### CODE REVIEW

**Review by Oluwafayokemi — APPROVED** [review]
Nice Work @Veraclins !

**Review by yomigeek — APPROVED** [review]
NIce work! Thanks for working on the fix @Veraclins

File: server/v1/helpers/queryHelper.js [hunk 1]
```diff
@@ -17,7 +17,13 @@ export default {
       { model: User, as: 'author', attributes: ['username', 'firstName', 'lastName', 'bio', 'image'] },
       { model: Rating, as: 'ratings', attributes: ['id', 'userId', 'value', 'createdAt', 'updatedAt'] },
       { model: Tag, as: 'tags', attributes: ['id', 'articleId', 'title', 'createdAt', 'updatedAt'] },
-      { model: Comment, as: 'comments', attributes: ['id', 'body', 'createdAt', 'updatedAt'] },
+      {
+        model: Comment,
+        as: 'comments',
+        include: [
+          { model: User, as: 'commenter' },
```
Comment (madeofhuman) [inline server/v1/helpers/queryHelper.js]: NIce work including the commenter as it would make displaying the data on the client much easier.

**Review by madeofhuman — APPROVED** [review]
Good work, @Veraclins

**Review by tonymontaro — APPROVED** [review]
Nice work.

**Pull request merged**